### PR TITLE
fix: comment mention email link

### PR DESCRIPTION
**Pull request by sshanzel**

### CODE REVIEW

File: src/schema/posts.ts [hunk 1]
```diff
@@ -449,6 +449,9 @@ export const reportReasons = new Map([
 export const getPostPermalink = (post: Pick<GQLPost, 'shortId'>): string =>
   `${process.env.URL_PREFIX}/r/${post.shortId}`;
 
+export const getPostPageLink = (post: Pick<GQLPost, 'shortId'>): string =>
```
Comment (idoshamun) [inline src/schema/posts.ts:452]: I think we have it already somewhere in the codebase

**Review by idoshamun — CHANGES_REQUESTED** [review]
Please check my comment :)

File: src/schema/posts.ts (hunk 1 above)
Comment (rebelchris) [reply src/schema/posts.ts:452]: commentsPermalink I think from the top of my head

Comment (sshanzel) [reply src/schema/posts.ts]: You're right, Chris! Thank you for the reference!

**Review by idoshamun — APPROVED** [review]
Ok Ok, let's roll! 🚢

**Review by rebelchris — COMMENTED** [review]
Nice! 💖

**Pull request merged**